### PR TITLE
Add support for 'get_moves` action

### PR DESCRIPTION
**Pull request by claytono**
This adds a new `get_moves` action which will scroll down to the moves part of the screen using a swipe input, trigger calcyiv, and then scroll back up to the top.  The scroll up swipe is slightly longer than the scroll down, because I found that it didn't always scroll completely to the top otherwise.

Note that the config.yaml.example coordinates are for a pocophone and probably don't work on a 1080p phone.

### CODE REVIEW

File: ivcheck.py [hunk 1]
```diff
@@ -127,14 +127,23 @@ def __init__(self, args):
...
-    async def swipe(self, location, duration):
-        await self.p.swipe(
-            self.config['locations'][location][0],
-            self.config['locations'][location][1],
-            self.config['locations'][location][0],
-            self.config['locations'][location][1],
-            duration
-        )
+    async def swipe(self, location, duration, reverse=False):
+        if len(self.config['locations'][location]) == 2:
```
Comment (esauvisky) [inline ivcheck.py]: Is this if block for a tap&hold/long-tap behaviour (the one we simulate with swipes because of the lack of embedded support)?

**Comment by esauvisky** [comment]
Hey, @claytono, I don't understand the whole `reverse=True` thing. Where is it being used? If we're using two different locations in the yaml, one for scrolling down and another up, it seems kinda pointless?

Also, check my comment on incoming line 131: at first glance, that looks like a "tap&hold" feature that's simulated via a swipe using the same coordinates for 'start' and 'end'. Does this PR requires that functionality in order to work? In case it doesn't I'd say it should get it's own PR rather than sneakily getting though this one. Do you guys agree?

**Comment by claytono** [comment]
I've updated this to only scroll down if calcyiv isn't reporting the fast and charge moves already.  This also fixes the regex used for parsing the special move, since calcyiv is sending both the first and second charge move.

**Comment by esauvisky** [comment]
Why did you close the PR? Well, I merged it on my fork anyway :grin:

**Pull request closed without merging**